### PR TITLE
feedback rate limiting on home page

### PR DESCRIPTION
**Pull request by cpapazian**
https://github.com/18F/myusa/issues/470#issuecomment-69979991

### CODE REVIEW

File: app/models/feedback.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  private
+
+  def rate_limit_per_5_seconds
+    if Feedback.where(remote_ip: self.remote_ip).last_5_seconds.exists?
+      errors.add(:base, :rate_limit_per_5_seconds)
+    end
+  end
+
+  def rate_limit_per_day
+    if Feedback.where(remote_ip: self.remote_ip).last_24_hours.count >= 10
```
Comment (yozlet) [inline app/models/feedback.rb]: Feels like a very low limit, given how some IP addresses are shared by many machines.

How about we:
* put the limit constant on a config
* report to NewRelic whenever the limit is reached

Comment (polastre) [inline app/models/feedback.rb]: Big :+1: to both suggestions!

**Comment by cpapazian** [comment]
not catching the error will allow new relic to do its thing automagically.

**Comment by cpapazian** [comment]
@yozlet

**Comment by polastre** [comment]
:+1:

**Comment by cpapazian** [comment]
@yozlet can  you take a look so we can merge this?

**Comment by polastre** [comment]
:shipit:

**Pull request merged**